sites/public/lib/helpers.tsx

```diff
@@ -163,7 +163,9 @@ export const getListings = (listings) => {
 }
 
 export const usToIntlPhone = (usPhoneNumber: string): string => {
```

Comment:
Since `null` is now a possible return of `usToIntlPhone`, can you please update the return type to reflect that.